src/nvim/mouse.c

```diff
@@ -246,6 +246,16 @@ static int get_fpos_of_mouse(pos_T *mpos)
   return IN_BUFFER;
 }
 
+void do_move_autocmd(int screenrow, int screencol) {
+	Dictionary data = ARRAY_DICT_INIT;
+	PUT(data, "screenrow", INTEGER_OBJ(screenrow));
```

Comment:
I don't think there's a way to call mousepos() from here, so passing the screenrow and screencol should do for now.

Reply:
Currently getting exactly what I needed out of this and it appears to be working fine, so gonna mark as ready for review.